frontend/src/components/header-logo.tsx

```diff
@@ -78,43 +87,35 @@ const HeaderLogo = () => {
     const textColor = useColorModeValue('white', 'black');
 
     return (
-        <LinkBox
-            p="1.05rem"
-            bg={bg}
-            shadow="lg"
-            data-cy="header-logo"
-            borderRadius="0.5rem"
-            boxShadow="0 10px 20px 0 rgba(0, 0, 0, 0.1)"
-            mr="0rem"
-        >
-            <Flex display={{ base: 'none', md: 'block' }}>
-                <Image src={bigLogo} alt="logo" width={260} height={77} />
-            </Flex>
-            <Flex display={{ base: 'block', md: 'none' }}>
-                <Image src={smallLogo} alt="logo" width={90} height={90} />
-            </Flex>
-            {logoAcc && <LogoAccesory iconSrc={logoAcc} h={40} w={40} />}
-            <NextLink href="/" passHref>
-                <LinkOverlay>
-                    <Flex position="absolute" bottom="-1rem" left="5%" w="20rem">
-                        <Text
-                            bg={textBg}
-                            pb="0.1rem"
-                            pt="0.2rem"
-                            px="1rem"
-                            align="left"
-                            color={textColor}
-                            fontSize="md"
-                            noOfLines={1}
-                            suppressHydrationWarning
-                            borderRadius="0.5rem"
-                            boxShadow="0 10px 20px 0 rgba(0, 0, 0, 0.1)"
-                        >
-                            {randomHeaderMessage()}
-                        </Text>
-                    </Flex>
-                </LinkOverlay>
-            </NextLink>
+        <LinkBox>
```

Comment:
fungerer ikke å trykke på logoen for å komme seg hjem på mobil

Reply:
Haha det ser jeg. Er ikke lagt til link der.